### PR TITLE
ACE Tracer adjustment, fix nighttime visibility, add white tracers

### LINKED ISSUE #2082: ACE tracers are too dim at night
ACE3 Version: 3.2.1

Mods:
@cba_a3
@ace3

Placed ACE3 Modules:
none

Description:
ACE tracers are too dim at night. Screenshots:
http://i.imgur.com/fecIInX.jpg
http://i.imgur.com/PpwYutO.png
My bloom setting is 0. 

Steps to reproduce:
Spawn any unit with tracers and shoot at night time. Ghosthawk is ok for this.

Where did the issue occur?
Editor

RPT log file:
none

### PR DESCRIPTION
**Pull request by Drofseh**
**When merged this pull request will:**
* Allow the optional ACE tracers to function properly at night, fix #2082 
All tracer rvmat's values have been changed to be closer or the same as vanilla tracer rvmat values for the same tracer colour. The new values for the rvmat's aren't perfect, but they work. day or night.
- ace_shell_tracer_xx models are now the same as the small arms tracers but have been scaled up 2x.
These look a lot better than the old shell tracers which were large and got very fuzzy when viewed with magnification or in fog.
~~ace_shell_tracer_red is now a copy of ace_shell_tracer_yellow, but using the red texture and rvmat.
This is because it had a completely different model then the other ace_shell_tracer_xx models that was large, fuzzy, and translucent. it made red tank tracers look like a big red cloud flying through the air.~~
- Added white tracer textures (copied from the yellow textures, desaturated, and slightly brightened), rvmats (similar to yellow with different RGB values), and models (copied from the yellow models but using the new white textures and rvmats) for bullets and shells. ~~nothing in vanilla uses white (I think) but I know some other mods do.~~
- Sets ammo that uses white vanilla tracers to use the new white ACE tracers instead of red.
- adds ace tracers to a couple of ammo classes added in Apex. If I notice more I'll add them as well.

Testing in Bright, Medium, and Low Light, and Complete Darkness
https://www.youtube.com/watch?v=XzbPDCpDFQI

Testing in Low Light with ACE Gen3 NVGs
https://www.youtube.com/watch?v=-eClU9pyjl4

Test classes can be uncommented and built, all based on the vanilla M249
They can be completely removed for release if desired.

Test version available here:
https://drive.google.com/open?id=1Rr7tPsyVrad8xQYG_M8QcBwUDXEvEZIa

- close #6633

### CODE REVIEW

File: AUTHORS.txt [hunk 1]
```diff
@@ -58,6 +58,7 @@ Dimaslg <dimaslg@telecable.es>
 dixon13 <dixonbegay@gmail.com>
 Drill <drill87@gmail.com>
 Dudakov aka [OMCB]Kaban <dudakov.s@gmail.com>
+Drosfeh <drofseh@gmail.com>
```
Comment (Dystopian) [inline AUTHORS.txt]: `sf` or `fs`? :)

Comment (bux) [reply AUTHORS.txt]: damn ... you have some good mistake tracking skillZ 🎉

**Comment by jonpas** [comment]
Should be noted: this is the optional ball tracers, not additional colors of normal tracers.

File: AUTHORS.txt (hunk 1 above)
Comment (Drofseh) [reply AUTHORS.txt]: lol, it is `fs` thanks

**Comment by Drofseh** [comment]
>Should be noted: this is for optional ball tracers, not additional colors of normal tracers.

I updated the PR comment to make that clearer.

**Comment by Drofseh** [comment]
My unit has been playing with this fix for the last three months with no problems.

I'd love it if this PR could get it's reviewed and merged.

**Comment by Drofseh** [comment]
I've noticed some vanilla ammo classes DO use a white tracer `\A3\Weapons_f\Data\bullettracer\tracer_white`

Currently these are largely set up to use the red Ace tracers.

Do you think these should be switched to white Ace tracer I made in order to remain consistent with their vanilla colouring, or should I leave them as they are?

**Comment by Drofseh** [comment]
>it does seem weird that we change the base to be red

I can see it being done before because there was no white Ace tracer and red is a much more common tracer colour, but since I've made one for this PR it seems reasonable to change them back to white.

**Comment by Neviothr** [comment]
@jaynus - review?

File: optionals/tracers/data/tracer_white.rvmat [hunk 2]
```diff
@@ -0,0 +1,18 @@
+////////////////////////////////////////////////////////////////////
```
Comment (bux) [inline optionals/tracers/data/tracer_white.rvmat]: comment needed?

**Comment by bux** [comment]
lgtm

File: optionals/tracers/data/tracer_white.rvmat (hunk 2 above)
Comment (Drofseh) [reply optionals/tracers/data/tracer_white.rvmat]: I copied it all from the yellow rvmat, but they all have the mikero tools comment left in.
Don't know why they were left in, but i'm happy to remove them if you think that's best.

Comment (jokoho48) [reply optionals/tracers/data/tracer_white.rvmat]: yes they can get removed

**Comment by Drofseh** [comment]
I wasn't satisfied with the large calibre shell tracers, so I've redone them by simply scaling the small arms tracer models up by 2x.
These shell tracers look a lot better now and are still large enough to be noticeably larger in game without being overpoweringly huge.

**Comment by dedmen** [comment]
You forgot the model.cfg entries for the new tracers.
I'll fix in my armake fix PR.

**Pull request merged**